crypto/ct/ct_vfy.c

```diff
@@ -219,7 +219,7 @@ int SCT_verify(const SCT_CTX *sctx, const SCT *sct)
     if (sct->log_id_len != sctx->pkeyhashlen ||
         memcmp(sct->log_id, sctx->pkeyhash, sctx->pkeyhashlen) != 0) {
         CTerr(CT_F_SCT_VERIFY, CT_R_SCT_LOG_ID_MISMATCH);
-        return 0;
+        return -1;
```

Comment:
The only distinction really was that the errors causing -1 to be returned were likely programmer error, whereas the errors causing 0 to be returned were likely due to legitimate input. That's not something that calling code should distinguish though, since it really shouldn't be trying to handle programmer error, so I'll simply this to just 0 or 1.